llvm/docs/CommandGuide/lit.rst

```diff
@@ -151,6 +151,11 @@ EXECUTION OPTIONS
  feature that can be used to conditionally disable (or expect failure in)
  certain tests.
 
+.. option:: --skip-test-time-recording
+
+ Disable tracking the wall time individual tests take to execute. This allows
+ external build systems to orchestrate the scheduled tests.
```

Comment:
I'm not sure the second sentence is strictly necessary. In the future, other use-cases might come up and the sentence might become redundant or even false.